hotel-management/src/components/Dropdown/index.tsx

```diff
@@ -0,0 +1,11 @@
+import { ReactElement } from 'react';
+
+import './index.css';
+
+type DropdownProps = {
+  isDropdownActive: boolean;
+  children: ReactElement;
+};
+
+export const Dropdown = ({ isDropdownActive, children }: DropdownProps) =>
```

Comment:
This component should manage isDropdownActive by itself. Imaging u have multiple dropdown in one place, so u have to create number of state to match